packages/website/src/pages/foundations/Iconography.tsx

```diff
@@ -1,34 +1,102 @@
+            <thead className="mod-no-border-top">
+                <tr>
+                    <th>Icon</th>
+                    <th>Name</th>
+                </tr>
+            </thead>
+            <tbody>
+                {variants.map((svgName) => {
+                    const SvgComponent = Icons[svgName];
+                    return SvgComponent ? (
+                        <tr key={svgName}>
+                            <td className="mod-no-border-bottom">
+                                <SvgComponent
+                                    className={classNames('mod-black', {
+                                        'mod-16': svgName.includes('16Px'),
+                                        'mod-24': svgName.includes('24Px'),
+                                        'mod-32': svgName.includes('32Px'),
+                                        'mod-48': svgName.includes('48Px'),
+                                        'mod-56': svgName.includes('56Px'),
+                                        'mod-64': svgName.includes('64Px'),
+                                    })}
+                                />
+                            </td>
+                            <td className="mod-no-border-bottom">{svgName}</td>
+                        </tr>
+                    ) : null;
+                })}
+            </tbody>
+        </table>
+    </div>
 );
 
-const IconItem = ({svgName, fileName}: {svgName: SvgName; fileName: string}) => (
-    <li className="cursor-pointer">
-        <LinkSvg svg={{svgName}} url={getSvgURL(fileName)}>
-            <label className="icon-name cursor-pointer">{svgName}</label>
-        </LinkSvg>
-    </li>
+const IconsTable: React.FunctionComponent = () => (
+    <TableWithFilter
+        id="iconography"
+        data={iconsList}
+        className="plasma-page-layout__section full-content"
+        tbodyClassName="icon-grid"
+        renderBody={(data: typeof iconsList) =>
+            data.map(({iconName, variants}) => (
+                <IconSetCard key={iconName} iconName={iconName} variants={variants as Array<keyof typeof Icons>} />
+            ))
+        }
+        filterBlankslate={{
+            title: <span className="h4-book mb1 nowrap">No icons match the applied filter</span>,
+        }}
+    />
 );
 
-const getSvgURL = (fileName: string) =>
-    `https://github.com/coveo/plasma/blob/master/packages/style/resources/icons/svg/${fileName}`;
+const code = `
+    import * as React from 'react';
+    import { InfoSize32Px } from '@coveord/plasma-react-icons';
+
+    export default () => <InfoSize32Px className="mod-info mod-72" />;
+`;
+
+export const IconographyExamples = () => (
+    <>
+        <PageHeader
+            section="Foundations"
+            title="Iconography"
+            thumbnail="iconography"
+            description="Icons are used to visually represent actions, functionalities, and features."
+        />
+        <div className="palsma-page-layout">

```

Comment:
I don't know if we have the same typo elsewhere?
```suggestion
        <div className="plasma-page-layout">
```